### PR TITLE
Fix project service dependency

### PR DESCRIPTION
**Pull request by umeshkumhar**
- Fix project service dependency
- Switch cloudbuild test to create_network

### CODE REVIEW

File: modules/kuberay-monitoring/main.tf [hunk 1]
```diff
@@ -14,7 +14,8 @@
 
 # Temporary workaround to ensure the GMP webhook is installed before applying PodMonitorings.
 resource "time_sleep" "wait_for_gmp_operator" {
-  count           = var.autopilot_cluster ? 1 : 0
+  ## Temporary workaroud, This is impacting for the standard cluster dependency graph
+  #count           = var.autopilot_cluster ? 1 : 0
```
Comment (imreddy13) [inline modules/kuberay-monitoring/main.tf:18]: why change this?

Comment (andrewsykim) [reply modules/kuberay-monitoring/main.tf:18]: I'm in favor of just removing this for now, it doesn't seem to impact the AP flake at all anyways

Comment (imreddy13) [reply modules/kuberay-monitoring/main.tf:18]: fyi @artemvmin

Comment (imreddy13) [reply modules/kuberay-monitoring/main.tf:18]: ok to do in followup PR

Comment (andrewsykim) [reply modules/kuberay-monitoring/main.tf:18]: https://github.com/GoogleCloudPlatform/ai-on-gke/issues/502

**Review by imreddy13 — APPROVED** [review]

**Pull request merged**